### PR TITLE
[WIP] fix-for-issue-4113

### PR DESCRIPTION
**Pull request by 1160200515**
Refs #4113

----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

**Comment by 1160200515** [comment]
I got it thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 
 
 ### Fixed
+- We fixed an issue about improving check for page numbers.
+(https://github.com/JabRef/jabref/issues/4113)
```
Comment (stefan-kolb) [inline CHANGELOG.md]: No newline here, please.

**Comment by tobiasdiez** [comment]
It appears that something went wrong and some other commits from master were added to this PR. Can you please rebase/merge the master so that only your changes remain. Thanks.

**Comment by 1160200515** [comment]
I am sorry I don't understand. what should I do ?

**Comment by tobiasdiez** [comment]
Have a look at the diff here in github. There is a lot of code that is marked as changed although you didn't touch it. These should vanish when use `git pull origin/master` and then `git push`.

**Comment by 1160200515** [comment]
I am sorry that I am not familiar with git, so what should I do?

**Comment by LinusDietz** [comment]
The problem is that this PR contains all kinds of commits that are unrelated to this PR. @1160200515 It would be great if you could fix this. Probably the easiest thing would be to do this in another branch?

If this is not possible we will have to close this PR, which would be kinda sad, since this would mean that your contribution will not be included into JabRef.

**Comment by tobiasdiez** [comment]
@1160200515 can you please remove the unrelated commits (maybe a simple git pull upstream master is enough), because otherwise we cannot review and include this PR. Please also add tests.

**Comment by tobiasdiez** [comment]
Superseeded by #4498

**Pull request closed without merging**